QUEST_LV_0100.tsv

```diff
@@ -9579,14 +9579,14 @@ QUEST_LV_0100_20151001_009574	{memo X}You have to save the mysterious girl while
 QUEST_LV_0100_20151001_009575	{memo X}You've obtained the stone!
 QUEST_LV_0100_20151001_009576	You couldn't obtain anything!
 QUEST_LV_0100_20151016_009577	
-QUEST_LV_0100_20151016_009578	
-QUEST_LV_0100_20151016_009579	
+QUEST_LV_0100_20151016_009578	$You mean you want to enter the Crystal Mines to find the evidence of Salvation? {nl}
+QUEST_LV_0100_20151016_009579	$So we are not in the situation to spare our troops for guarding Revelators while they check the evidence of salvation. {nl}But we also can't go against the order of the Bishop and Knight Commander either. {nl}
 QUEST_LV_0100_20151016_009580	
 QUEST_LV_0100_20151016_009581	
 QUEST_LV_0100_20151016_009582	
 QUEST_LV_0100_20151016_009583	
 QUEST_LV_0100_20151016_009584	
-QUEST_LV_0100_20151016_009585	
+QUEST_LV_0100_20151016_009585	$Anyway, the bishop himself also had a dream while praying in closure. {nl}In his dream, the Goddess told him the evidence of salvation will be found in the Crystal Mine. {nl}
```

Comment:
There are more bishops. For instance, Maven is considered a bishop, although an archbishop.
In Orsha there is Bishop Urbonas.